.github/cloud-samples-tools/test/affected_test.go

```diff
@@ -0,0 +1,43 @@
+package test
+
+import (
+	"cloud-samples-tools/pkg/utils"
+	"path/filepath"
+	"testing"
+)
+
+func TestChangedRoot(t *testing.T) {
+	config := utils.Config{PackageFile: []string{"package.json"}}
+	diffs := []string{
+		filepath.Join("testdata", "file.txt"),
+	}
+	got := utils.Changed(config, diffs)
+	expected := []string{"."}
+	equals(t, expected, got)
+}
+
+func TestChangedPackage(t *testing.T) {
+	config := utils.Config{
```

Comment:
issue: use a table-based / parameterized testing strategy. Here's an example:
https://github.com/googleapis/cloud-bigtable-cbt-cli/blob/main/cbt_test.go#L127

I think that we should test a wide variety of cases, both succeed and failure cases in these tests.

Reply:
Thanks! I was wondering if there was a recommended less repetitive way to test.